src/SFA.DAS.EmployerAccounts/Services/PayeSchemesService.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using SFA.DAS.EmployerAccounts.Data;
+using SFA.DAS.EmployerAccounts.Interfaces;
+using SFA.DAS.EmployerAccounts.Models.PAYE;
+using SFA.DAS.HashingService;
+using SFA.DAS.Validation;
+
+namespace SFA.DAS.EmployerAccounts.Services
+{
+    public class PayeSchemesService : IPayeSchemesService
+    {
+        private readonly IPayeRepository _payeRepository;
+        private readonly IEnglishFractionRepository _englishFractionRepository;
+        private readonly IHashingService _hashingService;
+
+        public PayeSchemesService(
+            IPayeRepository payeRepository,
+            IEnglishFractionRepository englishFractionRepository,
+            IHashingService hashingService
+            )
+        {
+            _payeRepository = payeRepository;
+            _englishFractionRepository = englishFractionRepository;
+            _hashingService = hashingService;
+        }
+        public async Task<IEnumerable<PayeView>> GetPayeSchemsWithEnglishFractionForHashedAccountId(string hashedAccountId)
```

Comment:
The method looks like it could do with some tidy up refactoring.

Reply:
Just copied and pasted, will take a close look.

Reply:
Done quick extract method... more could be done in that section dealing with English Fraction...

depends how much we want to change as part of this..let me know.